doc/cla/corporate/medoo.md

```diff
@@ -0,0 +1,15 @@
+China, 2022-04-05
+
+Medoo Tech agrees to the terms of the Odoo Corporate Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+jlzhou zh@medoo.tech https://github.com/medoo
```

Comment:
Also change this line the same way

Reply:
You closed the PR instead of doing the change? Where should I merge it?

Reply:
Sorry for inconvenient, I created brand new shining PR #88187 for this. :)